docs-chef-io/content/server/upgrade_server.md

```diff
@@ -14,43 +14,37 @@ aliases = ["/upgrade_server.html", "/upgrade_server/"]
     weight = 60
 +++
 
-## Chef Infra Server Upgrade Matrix
+Each new release of Chef Infra Server improves reliability and updates 3rd party components to ensure the security of the server. It is important to keep Chef Infra Server up to date in order to ensure the secure and reliable operation of Chef Infra in your organization.
 
-| Running Version | Upgrade Version | License | Version Support |
+{{< warning >}}
+Before upgrading a production server make sure to upgrade a test server to confirm the process.
+{{< /warning >}}
+
+## Upgrade Matrix
+
+If running a Chef Infra Server 12.17.15 or later you can upgrade directly to the latest releases of Chef Infra Server 14. If you are running a release prior to 12.17.15 you must perform a stepped upgrade as outlined below.
+
+| Running Version | Upgrade To Version | Requires License | Supported Version |
 |---------|---------|------|-----------|
 | 13 | 14 | Yes | Yes |
-| 12.17.15 | 14 | Yes | Yes |
+| 12.17.15 | 14 | Yes | No |
```

Comment:
Des the Supported Version column indicate if the Running Version is supported?

Reply:
I'm not sure if I'm following your meaning. The docs don't know what version of Chef Infra Server is running, although that would be a cool feature.